docker/nginx-custom.conf

```diff
@@ -0,0 +1,38 @@
+server {
+    listen       8080;
+    # same config for localhost, dev and prod environments
+    server_name  localhost 
+                swedgene.scilifelab-2-dev.sys.kth.se
+                genomes.scilifelab.se;
+    absolute_redirect off;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+    }
+
+    error_page  404              /404.html;
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
+        root   /usr/share/nginx/html;
+    }
+
+    # Cache media files 
+    location ~* \.(webp|svg|jpg|jpeg|png|gif)$ {
+        root   /usr/share/nginx/html;
+        expires 30d;
+        add_header Cache-Control "public";
+    }
+    # Cache font files 
+    location ~* \.(woff|woff2|ttf|eot|otf)$ {
+        root   /usr/share/nginx/html;
+        expires 30d;
+        add_header Cache-Control "public";
```

Comment:
I am a bit unsure about this but it may be a good thing also to specify max-age in the add_header
add_header Cache-Control "public, max-age=2592000";
since

Reply:
Thanks. I did some digging and it turns out [the `expires` header is not needed if `max-age` is included](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expires) so I've updated the config to use the max-age param instead in commit 7500e00953d8cfd91204e9ff65c5230533ecb59f